lib/manageiq/rack_attack.rb

```diff
@@ -0,0 +1,54 @@
+require 'rack/attack'
+
+module ManageIQ
+  module RackAttack
+    def self.enable
```

Comment:
This is the one place for the conditional logic to live.  I figured it was better to print when enabling Rack Attack so we can see when test/dev situations require us to disable it for more situations.